src/Tribe/Events/Views/V2/Models/Tickets.php

```diff
@@ -181,10 +181,13 @@ public function fetch_data() {
 
 						$number = number_format_i18n( $stock );
 
+						$ticket_label_singular = tribe_get_ticket_label_singular_lowercase( 'event-tickets' );
+						$ticket_label_plural = tribe_get_ticket_label_plural_lowercase( 'event-tickets' );
+
 						if ( 'rsvp' === $type ) {
 							$text = _n( '%s spot left', '%s spots left', $stock, 'event-tickets' );
 						} else {
-							$text = _n( '%s ticket left', '%s tickets left', $stock, 'event-tickets' );
+							$text = _n( "%s $ticket_label_singular left", "%s $ticket_label_plural left", $stock, 'event-tickets' );
```

Comment:
When we use a placeholder in translations we need to include the php comment describing what the placeholder is, like this: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#multiline-function-calls

Reply:
This won't work for translation, we want to use placeholders for the `$ticket_label_singular` and `$ticket_label_plural` as well and insert it in the `sprintf()` below.

Reply:
```suggestion
							$text = _n( "%1$s %2$s left", "%1$s %3$s left", $stock, 'event-tickets' );
```